### ISSUE TITLE
Card doesn't show the real status of AC

### ISSUE THREAD

**Issue by drobnjos** [KaSroka/Toshiba-AC-control#53]
I successfully installed the add-on and connected the Toshiba air conditioning. AC can be turned on, switch modes, change temperature. But the unit card does not show the current AC status correctly. If the AC is off and I turn it on using the controls on the Home Assistant card, the card does not change and still shows the AC status "off". However, the AC actually turns on and runs. Temperature control or mode change also work, but also do not show up on the card. So the AC responds, the change is reflected correctly in the Toshiba Home AC Control app, but the card still shows the "off" status in HA. It doesn't work to turn the AC off, maybe because the add-on "thinks" the AC is off?

HA log shows this error: 
WARNING (MainThread) [homeassistant.components.climate] Entity None (<class 'custom_components.toshiba_ac.climate.ToshibaClimate'>) implements HVACMode(s): off, auto, cool, heat, dry, fan_only and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/h4de5/home-assistant-toshiba_ac/issues

**Comment by KaSroka** [comment]
You can enable debug logs for toshiba-ac by adding this to your `configuration.yaml`:
```yaml
logger:
  default: warning
  logs:
    # log level toshiba-ac
    toshiba_ac: debug
    custom_components.toshiba_ac: debug
 ```
 
Either for some reason cloud is not sending the state updates or the format for your device is not correctly supported. Please post your findings (remove any sensitive info before posting) here so we can debug further.

**Comment by h4de5** [comment]
i think this issue arises when it switches to the self-cleaning mode.
i see the same message in the logs every now an then, but i cannot really trigger it. I guess I have to fix this in the ha-integration.

**Comment by drobnjos** [comment]
Hello, the issue was solved with the last update 2024.8.0. Integration works fine since than. Thank you.

I think that was the same problem as [https://github.com/h4de5/home-assistant-toshiba_ac/issues/186](url) or rather [https://github.com/KaSroka/Toshiba-AC-control/issues/50](url).
